### PR TITLE
chore: add TLS configs for keycloak

### PR DESCRIPTION
**Pull request by sauljabin**
@diablouma please test over this branch the dashboard, remember to point to https://localhost:8443

### CODE REVIEW

File: local-dev/setup.sh [hunk 1]
```diff
@@ -171,14 +187,42 @@ EOF
...
+
+"$WORK_DIR/issue-certificates.sh"
+
+if [ ${keycloak} = true ]; then
     setup_keycloak
-    ;;
-*)
+fi
+
+if [ ${kafka} = true ]; then
```
Comment (coltmcnealy-lh) [inline local-dev/setup.sh:226]: Please let's not do this, there is no use-case for setting up without kafka.

**Review by coltmcnealy-lh — APPROVED** [review]
Please revert the `--kafka` change. Everything else is fine.

**Comment by sauljabin** [comment]
This change modify the setup script for configuring keycloak with TLS and that way the dashboard can work

@diablouma please test the dashbord pointing to both
http://localhost:8888 and https://localhost:8443

And let me know if the change is needed

**Comment by sauljabin** [comment]
NOT NEEDED, master works well

**Pull request closed without merging**